lib/sessions/session-oauth.js

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
I agree, as I stated earlier the whole session/ dir is a mess so you can remove it or implemented as you prefer